[AGM-2295] Update AdGuard account KB (#621)
AdguardTeam/KnowledgeBase: docs/general/account/features.md

Reviewer: тут бы также слегка обновила название — AdGuard account с маленькой буквы. сейчас разнобой

## Manage account settings

> In the *Settings* section of your AdGuard account, you can update or reset your password, change your email address, enable one-time code log in or [two-factor authentication](../2fa), view the data stored about you by AdGuard and choose the language of your personal account and notifications.

Reviewer: ```suggestion
In the *Settings* section of your AdGuard account, you can update or reset your password, change your email address, enable one-time code login or [two-factor authentication](../2fa), view the data stored about you by AdGuard, and select the language of your account and notifications.
```

In this section, you can also manage your email preferences:

> - receive AdGuard’s monthly newsletters,

Reviewer: тут хочется поправить форматирование списков
плюс надо решить, что мы тут хотим сказать. Monthly newsletter — это наши дайджесты, просто newsletters — разные виды рассылок. Ну либо emails просто

```suggestion
- Receive AdGuard’s newsletters
```

> - receive transactional emails,

Reviewer: тут тоже правлю форматирование
```suggestion
- Receive transactional emails
```

- canceled.
You can also update your payment details.

> The devices bound to your AdGuard Ad Blocker licenses can be unbound or renamed. To unbind unused devices, click the number of bound devices next to *Devices*, then select the right arrow -> *Remove*.

Reviewer: сделала красивую стрелочку)
```suggestion
The devices bound to your AdGuard Ad Blocker licenses can be unbound or renamed. To unbind unused devices, click the number of bound devices next to *Devices*, then select the right arrow → *Remove*.
```
